### PR TITLE
More ec2 mappings

### PR DESCRIPTION
**Pull request by firenoo**
Continuing from the previous remapping PR https://github.com/GTNewHorizons/AE2FluidCraft-Rework/pull/115
* Updates buildscript (because it'll get squashed anyways...)
* Adds support for OpenComputers modules
* Ore dict bus now converts to export bus + ore dict card w/ settings retained
* Wireless now has proper NBT mappings

### CODE REVIEW

**Comment by firenoo** [comment]
Please don't merge this GIANT spotless apply until after review

**Review by eigenraven — APPROVED** [review]

**Pull request merged**
